challenge_manipulation/src/manipulation.py

```diff
@@ -682,7 +664,7 @@ def export_to_pdf(userdata):
             entities = [ e[0] for e in DETECTED_OBJECTS_WITH_PROBS ]
 
             # Export images (Only best MAX_NUM_ENTITIES_IN_PDF)
-            pdf.entities_to_pdf(robot.ed, entities[:MAX_NUM_ENTITIES_IN_PDF], "tech_united_manipulation_challenge")
+            # pdf.entities_to_pdf(robot.ed, entities[:MAX_NUM_ENTITIES_IN_PDF], "tech_united_manipulation_challenge")
```

Comment:
Ouch, reviews already paying off. 
I outcommented that stuff because it was giving me errors while testing. Something with xhtml.